crates/polars-core/src/chunked_array/ops/sort/arg_bottom_k.rs

```diff
@@ -0,0 +1,81 @@
+use polars_utils::iter::EnumerateIdxTrait;
+
+use super::*;
+
+#[derive(Eq)]
+struct CompareRow<'a> {
+    idx: IdxSize,
+    bytes: &'a [u8],
+}
+
+impl PartialEq for CompareRow<'_> {
+    fn eq(&self, other: &Self) -> bool {
+        self.bytes == other.bytes
+    }
+}
+
+impl Ord for CompareRow<'_> {
+    fn cmp(&self, other: &Self) -> Ordering {
+        self.bytes.cmp(other.bytes)
+    }
+}
+
+impl PartialOrd for CompareRow<'_> {
+    fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
+        Some(self.cmp(other))
+    }
+}
+
+pub fn _arg_bottom_k(
+    k: usize,
+    from_n_rows: usize,
```

Comment:
Small nit: This argument is redundant and could be get by `let height = by_column[0].len()`.